src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -322,7 +322,7 @@ const MountpointSuffix = ({ partition }: MountpointSuffixPropTypes) => {
           })
         );
       }}
-      aria-label="text input example"
+      aria-label="mountpoint suffix"
```

Comment:
can you please change it to "Mount point suffix text input" as it was in V1wizard?